pkgs/os-specific/linux/cpuset/default.nix

```diff
@@ -1,27 +1,37 @@
 { stdenv
 , fetchFromGitHub
-, python2Packages
+, pythonPackages
 }:
 
-python2Packages.buildPythonApplication rec {
+pythonPackages.buildPythonApplication rec {
   pname = "cpuset";
-  version = "1.5.8";
+  version = "1.6";
 
-  propagatedBuildInputs = [ ];
+  propagatedBuildInputs = with pythonPackages; [
+    configparser
```

Comment:
Not strictly necessary for Python 3, but if we want this package to continue to work with Python 2 this needs to be included.

Reply:
This package is usually used as a CLI tool so I don't think it's hugely important to keep the Python 2 support (especially that we're not testing it?), although I don't see much harm in it either.

Reply:
Yeah, me too, I am ok either way.